src/libmongoc/src/mongoc/mongoc-topology-description.c

```diff
@@ -1132,7 +1132,7 @@ mongoc_topology_description_invalidate_server (
    BSON_ASSERT (error);
 
    /* send NULL ismaster reply */
-   mongoc_topology_description_handle_ismaster (topology, id, NULL, 0, error);
+   mongoc_topology_description_handle_ismaster (topology, id, NULL, -1, error);
```

Comment:
This was not a bug before. Because error was always set, so the RTT was never updated. But I think -1 should consistently be the "invalid RTT" indicator.

Reply:
Cue earlier suggestion about using a constant.

Reply:
Did you miss replacing this with `MONGOC_RTT_UNSET`?

Reply:
Ah yes, fixed.